ttt_csharp/Board.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ttt_csharp
+{
+    public class Board
+    {
+        public List<object> Spots;
+        public Board()
+        {
+            Spots = new List<object>() {1, 2, 3, 4, 5, 6, 7, 8, 9};
+        }
+
+        public object GetSpot(int spot)
+        {
+            return Spots[spot - 1];
+        }
+
+        public void SetSpot(int spot, char marker)
+        {
+            Spots[spot - 1] = marker;
+        }
+
+        public List<object> GetAvailableSpots()
+        {
+            List<object> availableSpots = new List<object>();
+            foreach (var spot in Spots)
+            {
+                if (spot is int)
```

Comment:
`spot is int` can be a bit cryptic, I would extract this to a method perhaps called `IsAvailable(spot)`.

Reply:
extracted in https://github.com/olgaboiar/ttt_csharp/pull/1/commits/0df81a04b911b94ff8be42adcdd621eaecb5bf6c